src/core/config/Categories.json

```diff
@@ -53,7 +53,8 @@
             "To MessagePack",
             "From MessagePack",
             "To Braille",
-            "From Braille"
+            "From Braille",
+            "From Length Value"
```

Comment:
I'd rename this to something like 'LV Decode'. Then we can have an 'LV Encode' operation that does the opposite too.